crates/runtime/src/dataconnector/github.rs

```diff
@@ -89,6 +89,7 @@ impl GithubTableArgs for PullRequestTableArgs {
                             comments(first: 100) {{num_of_comments: totalCount}}
                             commits(first: 100) {{num_of_commits: totalCount, hashes: nodes{{ id }} }}
                             assignees(first: 100) {{ assignees: nodes {{ login }} }}
+                            updatedAt
```

Comment:
Move next to other timestamps and rename to snake case.

Reply:
fixed